src/api.ts

```diff
@@ -56,7 +55,17 @@ server.route({
       limit: string;
     };
     const limit = parseInt(query.limit);
-    const cursor = parseCursor(query.cursor);
+    let oldCursor = undefined;
+    if (query.cursor && query.cursor.includes("/")) {
+      // We don't need the date anymore, but can grab the index
+      const [, index] = query.cursor.split("/");
+      oldCursor = index;
+    }
```

Comment:
we can remove this after like a day right? this is to support people scrolling as we deploy?

Reply:
Yeah, isn't needed for long at all.